client/lib/feed-stream-store/actions.js

```diff
@@ -197,7 +197,7 @@ export function receiveGap( id, gap, error, data ) {
 
 export function dismissPost( id, post ) {
 	Dispatcher.handleViewAction( {
-		type: ActionType.DISMISS_FEED_STREAM_POST,
+		type: ActionType.DISMISS_POST,
```

Comment:
huh!

Reply:
I had left a previous comment about this but now I can't find it. It appears like this has been erroneous for a while in production. The constant `DISMISS_FEED_STREAM_POST` doesn't exist and I think that the only reason the app hasn't failed is because the `undefined` value it returns is unique among the feed stream actions…